### PR TITLE
Added note for depricated UDP

### PR DESCRIPTION
**Pull request by Joohansson**

### CODE REVIEW

File: docs/running-a-node/docker-management.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ docker run --restart=unless-stopped -d \
 | Option                                                | Purpose |
 |                                                       |         |
 | `-d`                                                  | Starts the docker container as a daemon |
-| `-p 7075:7075/udp`                                    | Maps the network activity port |
+| `-p 7075:7075/udp`                                    | Maps the network activity port (depricated since V21) |
```
Comment (zhyatt) [inline docs/running-a-node/docker-management.md:46]: depricated > deprecated

File: docs/running-a-node/docker-management.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ services:
     image: "nanocurrency/nano:${NANO_TAG}" # tag you wish to pull, none for latest
     restart: "unless-stopped"
     ports:
-     - "7075:7075/udp"   #udp network traffic
+     - "7075:7075/udp"   #udp network traffic (depricated since V21)
```
Comment (zhyatt) [inline docs/running-a-node/docker-management.md:161]: depricated > deprecated

**Review by zhyatt — CHANGES_REQUESTED** [review]
Thanks @Joohansson ! Just a couple minor spelling changes being requested.

File: docs/running-a-node/docker-management.md (hunk 1 above)
Comment (Joohansson) [reply docs/running-a-node/docker-management.md]: fixed

File: docs/running-a-node/docker-management.md (hunk 2 above)
Comment (Joohansson) [reply docs/running-a-node/docker-management.md]: fixed

**Review by zhyatt — APPROVED** [review]

**Pull request merged**